rest/revocation_test.go

```diff
@@ -1868,6 +1868,82 @@ func TestReplicatorRevocationsWithTombstoneResurrection(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestReplicatorRevocationsWithChannelFilter(t *testing.T) {
+	base.RequireNumTestBuckets(t, 2)
+
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll) // CBG-1981
+
+	// Passive
+	_, rt2 := InitScenario(t, nil)
+	defer rt2.Close()
+	rt2_collection := rt2.GetSingleTestDatabaseCollection()
```

Comment:
```suggestion
	rt2Collection := rt2.GetSingleTestDatabaseCollection()
```

go idioms state variables are not named with underscores but camelcase.